.github/workflows/examples-openiotsdk.yaml

```diff
@@ -106,6 +106,15 @@ jobs:
               run: |
                   scripts/run_in_build_env.sh './scripts/build_python.sh --install_wheel build-env'
 
+            - name: Install pyedmgr
+              timeout-minutes: 10
+              run: |
+                  scripts/run_in_build_env.sh 'git clone https://gitlab.arm.com/iot/open-iot-sdk/tools/pyedmgr'
+                  cd pyedmgr
+                  ../scripts/run_in_build_env.sh 'pip install -r requirements.txt'
+                  ../scripts/run_in_build_env.sh 'python setup.py install'
+                  cd ..
```

Comment:
Is the pyedmgr available as a Python package? Or not yet? If not, I think providing it directly to the development environment will be better. Then it can be useful for testing examples locally.

Reply:
ok, so we can't simply add it to Pigweed env... No, I'm thinking rather put this installation in our build script `openiotsdk_example.sh`. If you run the test action, then we activate the pigweed env, check if pyedmgr is available if not install it and execute the test. 
Then whether you do it in CI or locally you always have pyedmgr.

Reply:
Good idea, I will do that